README.md

```diff
@@ -461,6 +467,153 @@ However, there is one important restriction: you **must not decide to completely
+
+You can reconfigure the DB dump to include more or different sites and/or change the defaults. Read below to learn more.
+
+The multisite application will use `test-` prefixed collections in the MongoDB database to avoid losing local development data when running the tests. This is achieved by setting the `CI` environment variable to `1` when running the tests (see the `e2e:*` scripts in `package.json`).
+
+### Prerequisites
+
+Install [MongoDB Tools](https://docs.mongodb.com/database-tools/installation/installation-linux/#installation). You should validate that the `mongodump` and `mongorestore` commands are available in your terminal.
+
+### Running the tests
+
+You'll find example tests in the `cypress/test` folder. To run the tests:
+
+```bash
+# Start the multisite platform in production mode
+npm run e2e:serve
+# OR start the multisite platform in development mode (if you want to see the changes in real-time)
+npm run e2e:dev
+# In a new terminal window, run the tests in headless mode
+npm run e2e:run
+# OR run the tests in interactive mode
+npm run e2e:open
+```
+
+### Updating the Cypress configuration DB dumps 
+
+1. Remove all `test-` prefixed collections from your local MongoDB database.
+2. Add `admin` user to the dashboard site: `CI=1 node app @apostrophecms/user:add admin admin --site=dashboard`.
+3. Run `npm run e2e:dev` to start the multisite platform in development test mode.
+4. Open `http://dashboard.localhost:3000` in your browser, login with user `admin` and configure the sites you want for testing.
+5. In a new terminal window, run `CI=1 node app site:cypress-config --site=dashboard`. If you want to change the default configuration to be another site (it's the first in the list by default), you can pass the site shortname as an argument: `node app site:cypress-config site-demo --site=dashboard`.
+6. Copy the content of the terminal output between the `# cypress.config.js` and `# END cypress.config.js` comments to the `cypress.config.js` file, replacing the existing content. Feel free to update the root configuration options to match your needs (e.g., `viewportviewportWidth`, `viewportHeight`, etc.).
+7. Copy and execute the content of the terminal output between the `# DB dump commands` and `# END DB dump commandss` comments.
+
+> NOTE: if you are using a non-standard MongoDB connection string, you should update the `cypress.config.js` file and the DB dump commands accordingly.
+
+> NOTE: the script assumes that your admin API Key is named `cypressAPIKey`. If you are using a different name, you should update the `cypress.config.js` file accordingly.
+
+### Integrating Cypress in your existing projects
+
+1. Ensure that your project is fully configured, following the instructions in this documentation. This includes any port changes, theme configurations, and any other customizations you have made.
```

Comment:
I personally feel you should mention the `/etc/hosts` file here.

Reply:
I agree.

Reply:
"This section is for those who want to add Cypress testing to an existing project that was not created from a recent version of this starter kit and does not already contain the following updates."